### DISCUSSION TITLE
OrientDB Enterprice Agent for 2.2.13 version

### DISCUSSION THREAD

**Discussion by sviatoshkin** [orientechnologies/orientdb#9846] [Q&A] [answered]
If the EE version of the database has become free, can you please help find a jar agent for the orientdb version-2.2.13?

**Comment by tglman** [comment, marked as answer]
Hi,
The version 2.2.x has been out of support for years now, we do not have any open source version of the agent for that series of release, please use a more recent version like 3.2.x
